cmd/juju/application/bundle.go

```diff
@@ -647,6 +664,28 @@ func (h *bundleHandler) writeAddedResources(resNames2IDs map[string]string) {
 	}
 }
 
+// scaleApplication updates the number of units for an application.
+func (h *bundleHandler) scaleApplication(change *bundlechanges.ScaleChange) error {
+	// TODO: add verbose output for details
```

Comment:
Are you going to address this? What would you say in verbose that isn't in the normal output?

Reply:
cut and paste error